Cython/Compiler/FlowControl.py

```diff
@@ -717,14 +717,42 @@ def visit_FuncDefNode(self, node):
 
         for arg in node.args:
             self._visit(arg)
+
+        fastcall_directives = node.local_scope.directives.get('fastcall_args', [None, None])
```

Comment:
I think two separate variables would be better here than the less readable indexing below.